addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeHome.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+      
+    <thing-type id="NAWelcomeHome">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="netatmoapi" />
+        </supported-bridge-type-refs>
+        
+        <label>Welcome Camera Home</label>
+        <description>This represents a home</description>
+
+        <channels>
+            <channel id="welcomeHomeId" typeId="homeid"></channel>
+        	<channel id="welcomeHomeName" typeId="homename"></channel>
+        	<channel id="welcomeHomeCity" typeId="homecity"></channel>
+        	<channel id="welcomeHomeCountry" typeId="homecountry"></channel>
+        	<channel id="welcomeHomeTimezone" typeId="hometimezone"></channel>
+        	
+            <channel id="welcomeHomeSomebodyAtHome" typeId="homesomebodyathome"></channel>
+            <channel id="welcomeHomePersonCount" typeId="homepersoncount"></channel>
+            <channel id="welcomeHomeUnknownCount" typeId="homeunknowncount"></channel>
+                                            	    
+        </channels>
+        
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
```

Comment:
I get the welcomeHomeId from the netatmo api, don't know if its a good idea to make it possible to manually change it, whats the preferred implementation?

For some api calls, the welcomeHomeId is used so i get it back as channel to use it in scripts or rules, don't know if you can get this information there also from the parameters